deluge/scripts/create_plugin.py

```diff
@@ -238,7 +238,7 @@ def get_resource(filename):
 
 class GtkUI(GtkPluginBase):
     def enable(self):
-        self.glade = gtk.glade.XML(get_resource("config.glade"))
+        self.glade = Gtk.glade.XML(get_resource("config.ui"))
```

Comment:
Yea i changed some of the other parts, but ill give it a look through for anything remaining.